org.altlinux.Tuner.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.altlinux.Tuner
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: tuner
+add-extensions:
+  org.altlinux.Tuner.Plugin:
+    directory: extensions
+    merge-dirs: lib/tuner/plugins;share
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-config/dconf
```

Comment:
this should be `:ro`?


also requires an exception. https://docs.flathub.org/docs/for-app-authors/linter#exceptions